recipes/tacco/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "tacco" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tacco-{{ version }}.tar.gz
+  sha256: d5f65ac0a874229bc5c68bb6f1a6509c7dd3b5a19c57a8427093f726a215251b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools <=60
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.7
+    - requests
+    - importlib  # [py==26]
```

Comment:
I'd just remove this one to keep it noarch. conda-forge isn't building on Python <=3.6 by default anyways.

Reply:
good point, thanks! I adjusted it accordingly.

Reply:
I also removed the pip check as it claimed that mkl is not available while it is listed in all requirements and also in the installation schedule

Reply:
I don't think MKL is something that pip is capable of detecting.

Reply:
Well, finally got it.. Thanks for the help!